### ISSUE TITLE
Feature Request: A way to add links to charts

### ISSUE THREAD

**Issue by victorprens** [swimlane/ngx-charts#1146]
**I'm submitting a ...**  (check one with "x")

- [ ] bug report - search github for a similar issue or PR before submitting
- [x] feature request
- [ ] support request - use StackOverflow (add the `ngx-charts` tag) or the gitter chat for support questions

**Current behavior**
Currently, there's no way to add links to the names in charts.

**Expected behavior**
An input property that allows the charts components to receive links

**Reproduction of the problem**
![image](https://user-images.githubusercontent.com/14033988/56221029-1004f300-602f-11e9-8271-aaa1fe5fb9b4.png)

 e.g. in case of a horizontal bar chart, I would like to be able of adding links to the name of the countries.

**What is the motivation / use case for changing the behavior?**
Everytime you need to link chart data to a new component or maybe to other website, It's impossible.

**Please tell us about your environment:**

- WINDOWS 10
- IDE - Visual Studio Code
- NPM
- Node

* **ngx-charts version:** 9.0.0

* **Angular version:** 6.1.0

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by marjan-georgiev** [comment]
To clarify, do you mean the axis labels, or the legend? Clicking on the legend items emits an event that you can listen to and then use to navigate away. The axis labels don't have that yet.

**Comment by victorprens** [comment]
Yes @marjan-georgiev the axis labels are the ones that I want to add the feature.
